src/renderer/views/About/About.css

```diff
@@ -23,7 +23,7 @@
   display: grid;
   grid-gap: 16px;
   grid-template-columns: 1fr 1fr;
-  margin-block: 80;
+  margin-block: 80px;
```

Comment:
As the value was previously invalid, this wasn't getting used but now that we have turned it into a valid value we would be adding 80px more between the version number and tiles on the about page (there is already a 32px margin on the bottom of the version number so with this it would then be 112px).

In my mind it makes more sense to remove the margin but as other two other people have already clicked approve maybe the extra margin is desired?

Reply:
Oh you are right best to remove it